templates/python/custom-copilot-basic/src/config.py.tpl

```diff
@@ -0,0 +1,22 @@
+"""
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License.
+"""
+
+import os
+
+class Config:
+    """Bot Configuration"""
+
+    PORT = 3978
+    APP_ID = os.environ.get("BOT_ID", "")
+    APP_PASSWORD = os.environ.get("BOT_PASSWORD", "")
+    {{#useOpenAI}}
+    OPENAI_API_KEY = os.environ.get("OPENAI_API_KEY", "") # OpenAI API key
+    OPENAI_MODEL_DEPLOYMENT_NAME = os.environ.get("OPENAI_MODEL_DEPLOYMENT_NAME", "") # OpenAI model deployment name
+    {{/useOpenAI}}
+    {{#useAzureOpenAI}}
+    AZURE_OPENAI_API_KEY = os.environ.get("AZURE_OPENAI_API_KEY", "") # Azure OpenAI API key
```

Comment:
I'm not sure whether we want to give the config a default value. If it's required, maybe failing in the config class brings some benefit on troubleshooting?